### PR TITLE
presets: Change presets to match default config

### PR DESCRIPTION
**Pull request by nazrhom**
Change-type: major
Signed-off-by: Giovanni Garufi <giovanni@resin.io>

### CODE REVIEW

File: lib/presets.js [hunk 1]
```diff
@@ -42,6 +43,16 @@ const authenticate = () => {
   });
 };
 
+const isIncrementalCommit = (changeType) => {
+  return Boolean(changeType) && changeType.trim().toLowerCase() !== 'none';
```
Comment (hedss) [inline lib/presets.js]: Really should be parentheses around the second test.

**Comment by lurch** [comment]
Does changing the presets to match the default config mean that Versionist is now slightly _less_ non-opinionated ? :wink:

**Comment by nazrhom** [comment]
@hedss yup. Btw I had to revert the parenthesis in 
```
  return Boolean(changeType) && changeType.trim().toLowerCase() !== 'none';
```
because eslint was complaining

**Comment by nazrhom** [comment]
@resin-ci retest

**Comment by nazrhom** [comment]
@resin-ci retest

**Pull request merged**